core-feature-pack/src/main/resources/content/bin/common.ps1

```diff
@@ -141,14 +151,22 @@ Param(
 		if (-not(Test-Path $JBOSS_LOG_DIR)) {
 			$dir = New-Item $JBOSS_LOG_DIR -type directory -ErrorAction SilentlyContinue
 		}
-        $PROG_ARGS += "-verbose:gc"
-        $PROG_ARGS += "-XX:+PrintGCDetails"
-        $PROG_ARGS += "-XX:+PrintGCDateStamps"
-        $PROG_ARGS += "-XX:+UseGCLogFileRotation"
-        $PROG_ARGS += "-XX:NumberOfGCLogFiles=5"
-        $PROG_ARGS += "-XX:GCLogFileSize=3M"
-        $PROG_ARGS += "-XX:-TraceClassUnloading"
-        $PROG_ARGS += "-Xloggc:$JBOSS_LOG_DIR\gc.log"
+
+		$JAVA_9_PLUS = Check-For-Java9+
+
+		$PROG_ARGS += "-verbose:gc"
+		if ($JAVA_9_PLUS -eq $true)
+		{
+			$PROG_ARGS += "-Xlog:gc*:file=%JBOSS_LOG_DIR%\gc.log:time,uptimemillis:filecount=5,filesize=3M"
```

Comment:
Bumping the filesize to a larger value here (perhaps also exposed as an environment variable with a default) should probably get rid of @marekkopecky 's concern about the rotation?

Reply:
JDK8 use log rotation, I believe that log rotation is useful feature that can be useful also on JDK11. I believe that unlimited size could cause problems to WF users ... anyway, it would be regression of behaviour as well ...